src/core_plugins/kibana/public/dashboard/grid/dashboard_grid.js

```diff
@@ -58,7 +58,7 @@ function ResponsiveGrid({
       isResizable={true}
       margin={[MARGINS, MARGINS]}
       cols={DASHBOARD_GRID_COLUMN_COUNT}
-      rowHeight={100}
+      rowHeight={20}
```

Comment:
We have consts for everything else, it feels like maybe this should be one, too.